index.html

```diff
@@ -222,5 +222,37 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact-form" id="contact">
+        <div class="contact-text">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <div class="contact-area">
+            <form action="https://formspree.io/f/xvolllzo" method="post" id="frmContact">
+              <label for="name">
+                <input type="text" name="name" id="name" maxlength="30" class="contact-input" placeholder="Enter your names..."
+                required/>
+              </label>
+              <label for="email">
+                <input type="email" name="email" id="email" class="contact-input" placeholder="Enter your gmail..."
+                required/>
+              </label>
+              <label for="text">
+                <textarea name="text" id="text" maxlength="500" class="contact-input" placeholder="Enter text here" cols="23" rows="5"></textarea>
```

Comment:
- [ ] Great job having some form validations in place. The only bit that is left according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) is to give your text area a `required` attribute. Kindly add it.